gamearea.py

```diff
@@ -522,7 +531,16 @@ def divide_into_even_odd(self, number):
             divided_nums.append(even_number)
             remaining_sum -= even_number
         divided_nums.append(remaining_sum)
-        divided_nums[self.odd_cat_id - 1] += 1
+        print(divided_nums)
```

Comment:
I'm guessing you forgot to remove this after debugging.

Reply:
Yes, Thank you.

Also I was trying out something before and forgot to removed some parts of it commited it by mistake, removed them as well.